presentation/src/main/java/com/anytypeio/anytype/presentation/editor/render/DefaultBlockViewRenderer.kt

```diff
@@ -87,19 +87,23 @@ class DefaultBlockViewRenderer @Inject constructor(
                     isPreviousBlockMedia = false
                     when (content.style) {
                         Content.Text.Style.TITLE -> {
-                            mCounter = 0
-                            result.add(
-                                title(
-                                    context = context,
-                                    mode = mode,
-                                    block = block,
-                                    content = content,
-                                    focus = focus,
-                                    root = root,
-                                    details = details,
-                                    restrictions = restrictions
+                            val obj = details.getObject(id = context)
+                            if (obj?.layout == ObjectType.Layout.NOTE) {
```

Comment:
consider adding comment here: 
// Workaround for skipping title block in objects with note layout

Reply:
fixed